{{cookiecutter.project_slug}}/.github/workflows/linting.yml

```diff
@@ -46,17 +46,15 @@ jobs:
       - name: 🎨 Prettier
         working-directory: ./client
         run: npm run format:check
-      - name: Run ESLint
-        working-directory: ./client
```

Comment:
This was breaking linting. Introduced very recently: https://github.com/thinknimble/tn-spa-bootstrapper/pull/272